astro/src/content/quickstarts/quickstart-javascript-nuxt-web.mdx

```diff
@@ -0,0 +1,194 @@
+import LoginArchitectureWeb from '/src/components/quickstarts/LoginArchitectureWeb.astro';
+import NextSteps from '/src/components/quickstarts/NextSteps.astro';
+import {RemoteCode} from '@fusionauth/astro-components';
+import QuickstartTshirtCTA from '/src/components/quickstarts/QuickstartTshirtCTA.astro'
+
+<Intro technology={frontmatter.technology}
+       repositoryUrl="https://github.com/FusionAuth/fusionauth-quickstart-javascript-nuxt-web"/>
+
+## Prerequisites
+* [Node LTS](https://nodejs.org/en/download)
+* [Docker](https://www.docker.com): The quickest way to stand up FusionAuth. (There are [other ways](/docs/get-started/download-and-install)).
+* [git](https://git-scm.com/downloads/): Not required but recommended if you want to track your changes.
+
+## General Architecture
+
+<LoginArchitectureWeb />
+
+## Getting Started
+In this section, you'll get FusionAuth up and running, and configured with the ChangeBank application.
+
+### Clone The Code
+First off, grab the code from the repository and change into that directory.
+
+```
+git clone https://github.com/FusionAuth/fusionauth-quickstart-javascript-nuxt-web.git
+cd fusionauth-quickstart-javascript-nuxt-web
+```
+
+### Run FusionAuth Via Docker
+
+<DockerSpinup kickstartUri={frontmatter.codeRoot + "/kickstart/kickstart.json"} />
+
+### Create Nuxt Application
+
+In this section, you'll set up a basic Nuxt application with three pages. 
+
+1. Homepage
+1. Account - protected
+1. Makechange - protected
+
+Create a new application using the `npx`.
+
+```shell
+npx nuxi@latest init changebank
+```
+
+Make sure you are in your new directory `changebank`.
+
+```shell
+cd changebank
+```
+
+Install [@sidebase/nuxt-auth](https://sidebase.io/nuxt-auth/v0.6/getting-started/installation), which simplifies integrating with FusionAuth and creating a secure web application.
+
+```shell
+npm install next-auth
+```
+
+Copy environment variables from our complete application example.
+
+```shell
+cp ../complete-application/.env.example .env.local
+```
+
+Also copy an image file into a new directory within `public` called `img`.
+
+```shell
+mkdir ./public/img && cp ../complete-application/public/img/money.jpg ./public/img/money.jpg
+```
+
+## Authentication
+
+Update `nuxt.config.ts` to include the new `@sidebase/nuxt-auth` and include the provider type of `authjs` just like below. Please note if you would like to code with any of the pages prior to enabling authentication, just set `globalAppMiddleware` to false.
+
+<RemoteCode url={frontmatter.codeRoot + "/complete-application/nuxt.config.ts"}
+  lang="ts" />
+
+Nuxt 3 includes <a href="https://nuxt.com/docs/guide/directory-structure/server#server-routes" target="_blank">Route Handlers</a>, which are the preferred way to handle REST-like requests. In the `Changebank` application you can configure NextAuth.js FusionAuth's provider in a new route handler by creating a file within `/server/api/auth/[...].ts`.
```

Comment:
As not a Nuxt developer, the file name [...].ts is a little weird. If this is normal in Nuxt then leave, if not, maybe make another note that the file name contains the brackets and dots.